scripts/update-gz

```diff
@@ -21,8 +21,8 @@ if os.path.exists(locales_folder):
 os.mkdir(locales_folder)
 
 # Begin by checking if gzip is supported. If not - return schema.json
-htaccess_rules = f'''RewriteCond %{{REQUEST_URI}} !^/(schema|zotero-schema) 
-RewriteRule ".?" "-" [S=LINES_TO_SKIP]
+htaccess_rules = f'''RewriteCond %{{REQUEST_URI}} !^/schema 
+RewriteRule ".?" "-" [S=LINES_TO_SKIP,L]
```

Comment:
`L` isn't right here, though — that breaks the dataserver completely by skipping the main MVC redirects.

Reply:
I see what you mean - I removed it.

For some reason when I was testing it on my local dataserver setup, all non /schema requests reached index.php as they were supposed to even with that L flag. Probable can blame it on my local dataserver or apache setup